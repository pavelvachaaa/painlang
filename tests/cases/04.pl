for(let i: number = 0; i < 10; i++) {
    print(i);
}

let j: number = 0;
while (j < 10) {
    if(j <= 5) {
        print("mensi nebo rovno 5");
    } else {
        print("vetsi nez 5");
    }

    j++;
}

let z: number = 0;
for(let i: number = 0; i < 10; i++) {
    z += i;
}

print(z);
