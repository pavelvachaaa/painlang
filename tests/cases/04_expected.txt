0
1
2
3
4
5
6
7
8
9
mensi nebo rovno 5
mensi nebo rovno 5
mensi nebo rovno 5
mensi nebo rovno 5
mensi nebo rovno 5
mensi nebo rovno 5
vetsi nez 5
vetsi nez 5
vetsi nez 5
vetsi nez 5
45